src/app/components/transferDetailView/index.tsx

```diff
@@ -64,23 +117,71 @@ function TransferDetailView({
   address,
   hideAddress,
   hideCopyButton,
+  outputScript,
+  outputScriptIndex,
 }: Props) {
+  const { t } = useTranslation('translation', { keyPrefix: 'CONFIRM_TRANSACTION' });
+  const [showScriptOutput, setShowScriptOutput] = useState(false);
+
+  const onButtonClick = () => {
+    setShowScriptOutput(true);
+  };
+
+  const onCrossClick = () => {
+    setShowScriptOutput(false);
+  };
+
   return (
-    <RowContainer>
-      {icon && <Icon src={icon} />}
-      {amount ? (
-        <ColumnContainer>
-          <AmountText>{amount}</AmountText>
-          {children}
-        </ColumnContainer>
-      ) : (
-        <TitleText>{title}</TitleText>
+    <>
+      {showScriptOutput && (
+        <ShowScriptBackgroundContainer>
+          <CrossContainer onClick={onCrossClick}>
+            <TransparentButton>
+              <img src={Cross} alt="cross" />
+            </TransparentButton>
+          </CrossContainer>
+          <ScriptOutputHeadingText>{`${t(
+            'SCRIPT_OUTPUT',
+          )} #${outputScriptIndex}`}</ScriptOutputHeadingText>
+          <ScriptText>script: btc.Script.encode {'{['}</ScriptText>
+          {outputScript &&
+            outputScript.map((script) => {
+              if (script instanceof Uint8Array) {
+                return (
+                  <ScriptText>
+                    {`new TextEncoder().encode (${new TextDecoder().decode(script)})`},
+                  </ScriptText>
+                );
+              } else {
+                return <ScriptText>{script},</ScriptText>;
+              }
+            })}
+
+          <ScriptText>{']}'}</ScriptText>
```

Comment:
can we confirm this was the desired outcome? it was not clear to me from the linear issue comments

Reply:
Updated the PR after getting the confirmation about the script output from mahmoud. Added the recap of the discussion in linear issue https://linear.app/xverseapp/issue/ENG-2314#comment-11723000